cluster/src/main/java/org/apache/iotdb/cluster/config/ClusterDescriptor.java

```diff
@@ -48,17 +43,26 @@
 
   private static final Logger logger = LoggerFactory.getLogger(ClusterDescriptor.class);
   private static final ClusterDescriptor INSTANCE = new ClusterDescriptor();
-
-  private static final String OPTION_INTERVAL_META_PORT = "internal_meta_port";
-  private static final String OPTION_INTERVAL_DATA_PORT = "internal_data_port";
-  private static final String OPTION_CLUSTER_RPC_PORT = "cluster_rpc_port";
-  private static final String OPTION_SEED_NODES = "seed_nodes";
-
-  private ClusterConfig config = new ClusterConfig();
-  private static CommandLine commandLine;
+  private final ClusterConfig config = new ClusterConfig();
 
   private ClusterDescriptor() {
+    // copy needed configurations from the server's config to the cluster.
+    config.setClusterRpcPort(IoTDBDescriptor.getInstance().getConfig().getRpcPort());
```

Comment:
What is this line for? It seems that it is enough for us to make the default cluster port consistent with a single server.

Reply:
it seems the port in clusterConfig can be initialized with `IoTDBDescriptor.getInstance().getConfig().getRpcPort()` directly?

Reply:
Because `cluster_rpc+port` is copied from the `rpc_port `in the single module(iotdb-engine.properties), and the `loadprops()` method in ClusterDescriptor class are loaded properties in `iotdb- cluster.properities`, therefore, we need to fisrt set the `cluster_rpc_port`.

Besides the `rpc_port` of a single module can not be used directly, because when the debug mode(`open_server_rpc_port=true`) is turned on, the rpc_port of a single module may be different from that of a distributed cluster.